calculator/main.py

```diff
@@ -13,7 +13,19 @@ def create_new_calculator(operations=None):
     :param operations: Dict with initial operations.
                        ie: {'sum': sum_function, ...}
     """
-    pass
+
+    my_operations={}
+    for key in operations:
+        my_operations[key] = operations[key]
+    
+    calc = {
+        'operations':my_operations,
```

Comment:
Do you need `my_operations`?  What would be different if you just declared here `'operations': operations`?